gridcapa-swe-runner-app/src/main/java/com/farao_community/farao/swe/runner/app/services/FilesService.java

```diff
@@ -38,15 +38,19 @@ public FilesService(NetworkService networkImporter, MergingViewService mergingVi
     }
 
     public SweData importFiles(SweRequest sweRequest) {
-        Network network = networkService.importNetwork(sweRequest);
+
         MergingViewData mergingViewData = mergingViewService.importMergingView(sweRequest);
+        Network networkEsFr = networkService.importNetwork(sweRequest);
```

Comment:
To avoid repetition of conversion cgmes to xiidm (with events), I think it's better to call the import network once and save the url of xiidm file in Minio. Then import network from xiidm for each direction.

Reply:
Done I used Importers.loadNetwork () instead :)